internal/iam/command_ipgroup_update.go

```diff
@@ -0,0 +1,134 @@
+package iam
+
+import (
+	iamv2 "github.com/confluentinc/ccloud-sdk-go-v2/iam/v2"
+	"github.com/spf13/cobra"
+	"slices"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *ipGroupCommand) newUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update <id>",
+		Short: "Update an IP Group.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.update,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Update the Group Name of IP Group "ipg-12345"":`,
+				Code: `confluent iam ip-group update ipg-12345 --group_name "New Group Name"`,
+			},
+		),
+	}
+
+	pcmd.AddProviderFlag(cmd, c.AuthenticatedCLICommand)
+	cmd.Flags().String("group_name", "", "Name of the IP Group.")
+	cmd.Flags().StringSlice("cidr_blocks", []string{}, "List of CIDR blocks to replace existing CIDR blocks on the IP Group.")
```

Comment:
Maybe the doc can specify the relationship between these (e.g. something like "This parameter is not compatible with..."